identity/manager.go

```diff
@@ -0,0 +1,93 @@
+package identity
```

Comment:
Needs a brief description what this manager does. ex "maps ethereum network account to dto.Identity. currently creates a new eth account with PASSPHRASE password on CreateNewIdentity(). Used by Discovery Service."

Reply:
added